components/script/script_module.rs

```diff
@@ -651,12 +652,12 @@ impl ModuleTree {
 
             // 8-4.
             if child_parse_error.is_some() {
-                return child_parse_error;
+                errors.push(child_parse_error.unwrap());
             }
         }
 
         // Step 9.
-        return None;
+        return errors.iter().max().map(|err| err.clone());
```

Comment:
We should be able to do `errors.into_iter().max()` instead.

Reply:
Thanks for the hint!!! I finally understand the difference between `into_iter()` and `iter()` 😍 That's what I want!